### ISSUE TITLE
images not showing up in 2.2 technique's page that are there in 2.1

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3687]
If you load this [2.2 version of the Resize text](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html) SC, you'll see that the images for illustration 1 and 2  are broken
![image](https://github.com/w3c/wcag/assets/14298245/8c986464-0eca-4ac1-adae-6803aae066c8)
However, the same images work in [2.1 ](https://www.w3.org/WAI/WCAG21/Techniques/failures/F69.html)

I suspect this is not isolated to this page.

**Comment by patrickhlauke** [comment]
That first link should point to https://www.w3.org/WAI/WCAG22/Techniques/failures/F69

**Comment by hidde-logius** [comment]
I wasn't sure if this was unique to F69, so looked for more examples. Fwiw, I didn't find more examples of broken images, but I did find a few that worked (they were linked in the same way): [SCR32](https://www.w3.org/WAI/WCAG22/Techniques/client-side-script/SCR32.html), [C42](https://www.w3.org/WAI/WCAG22/Techniques/css/C42.html), [G56](https://www.w3.org/WAI/WCAG21/Techniques/general/G56.html), [G162](https://www.w3.org/WAI/WCAG21/Techniques/general/G162.html), [G207](https://www.w3.org/WAI/WCAG22/Techniques/general/G207.html).

**Comment by iadawn** [comment]
Fixed. The examples had changed from GIFs to PNGs but the files on the server had not been updated. I have checked that there are no other images that haven't been committed to the server. Couldn't find any.
